CONTRIBUTING.md

```diff
@@ -33,7 +33,7 @@ Pull requests are very welcome, but should be within the scope of the project, a
 - [`bazel`](https://bazel.build/)
 - [`docker`](https://www.docker.com/)
 
-You can build & test with `pnpm`:
+You can build & test with `pnpm`. At the moment version >= 9 is not supported:
```

Comment:
pnpm version 9 brings package lock to version 9, which seems to be unsupported by the bazel plugin used here.